drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java

```diff
@@ -184,14 +184,17 @@ public void testOperatorDropdownWhenOperatorCanBeSet() {
         page.operatorDropdown(widget -> {
             assertTrue(widget instanceof CEPOperatorsDropdown);
 
-            final CEPOperatorsDropdown operatorsDropdown = (CEPOperatorsDropdown) widget;
+            final CEPOperatorsDropdown dropdown = (CEPOperatorsDropdown) widget;
+
+            verify(dropdown.getBox()).setSelectedIndex(0);
```

Comment:
@jomarko My intention was not to test the value, in fact, I was focusing on the `setSelectedIndex ` method call. However, I changed the test a little bit to cover the case that you mentioned.